nwind/src/unwind_context.rs

```diff
@@ -96,6 +96,12 @@ impl< 'a, A: Architecture > UnwindHandle< 'a, A > {
 
         self.ctx.nth_frame += 1;
 
+        // avoid infinite loops
+        if self.ctx.nth_frame > 100 {
```

Comment:
```suggestion
        if self.ctx.nth_frame > 1000 {
```

Some code performs a deep recursion.